pkg/events/parse_args_helpers.go

```diff
@@ -208,7 +208,7 @@ func parseBpfHelpersUsage(arg *trace.Argument, helpersList []uint64) {
 			// helper number <i> is used. get its name from libbpfgo
 			bpfHelper, err := parsers.ParseBPFFunc(uint64(i))
 			if err != nil {
-				continue
+				usedHelpers = append(usedHelpers, fmt.Sprint(bpfHelper))
```

Comment:
Still need the continue after this line, otherwise `.String()` will be called on a uint64.

Reply:
ok will fix

Reply:
Just noticed that you are `bpfHelpers` as the raw value, it should be `i`.

Reply:
done